src/test/java/uk/gov/companieshouse/filinghistory/api/service/ValidatorFactoryTest.java

```diff
@@ -56,23 +55,22 @@ void shouldReturnAssociatedFilingValidatorForAnnotationKind() {
         // given
 
         // when
-        Validator<InternalFilingHistoryApi> actual = factory.getPutRequestValidator(InternalData.TransactionKindEnum.ASSOCIATED_FILING);
+        Validator<InternalFilingHistoryApi> actual = factory.getPutRequestValidator(TransactionKindEnum.ASSOCIATED_FILING);
 
         // then
         assertInstanceOf(AssociatedFilingPutRequestValidator.class, actual);
         assertEquals(associatedFilingPutRequestValidator, actual);
     }
 
-    // REMOVE TEST WHEN RESOLUTIONS ARE IMPLEMENTED
     @Test
```

Comment:
Keep this test, change exception.

Reply:
will break if not all enum cases are covered